_schemaV2.graphql

```diff
@@ -11783,7 +11783,34 @@ type Partner implements Node {
   ): LocationConnection
   meta: PartnerMeta
   name: String
+  partnerArtistsSearchConnection(
+    after: String
+    before: String
+    first: Int
+    last: Int
+    page: Int = 1
+    query: String!
+    size: Int = 10
+  ): partnerArtistsSearchConnection
```

Comment:
Should this type truly be `partnerArtistsSearchConnection`? The field/name can certainly be whatever (such as `partnerArtistsSearchConnection`), but the type appears to be a connection of artists. That is, a connection of edges where the `node` is an `Artist`. In that case, can the type of this field be the regular `ArtistsConnection` type? (https://github.com/artsy/metaphysics/blob/9ea046132dc846d2bfd4d560ae502c2b9337c3ba/src/schema/v2/artist/index.ts#L767).

Otherwise, we're sort of 'duplicating' types, by creating a new 'artist connection' type.